### ISSUE TITLE
Calculation issues no output despite the fact that no error message is issued

### ISSUE THREAD

**Issue by PPlessiez** [AntaresSimulatorTeam/Antares_Simulator#415]
**Description**
When launching the attached study (simulation for only MC year 348 as configured in the playlist), the simulation is apparently performed, but no output of the simulation is issued. The output folder only contains pre-processing data (like the ts-number folder). No error message is visible in the logs of the simulation.

**Screenshots and additional files**
Link to download the study: [Post'n'Go ](https://postngo.rte-france.com/?lang=fr&a=d&i=wkr1r9BRy9EoHymhEDse )

**OS and version**
 - OS: Windows
 - Antares Simulator version: 8.1

**Comment by flomnes** [comment]
Area where the simulation fails **VE_VHR_storage**. In this area, all thermal clusters are disabled.

Chronology of function calls:
1. `PartThermal::prepareAreaWideIndexes()`
2. `removeDisabledThermalClustersFromSolverComputations()`
3. `thermalTSNumberData::reset(const Study& study)`
4. `thermalTSNumberData::apply(Study& study)`

Step 1. resizes vector PartThermal::clusters to PartThermal::list

Step 2. removes all disabled clusters (all of them in this example). Object `list` is resized to 0, but object `clusters` **is not**.

Step 3. Uses clusters.size() to resize matrix `pTSNumberRules`.

Step 4. Uses list.size(), which is now 0

## TL;DR

One cluster list is resized to ignore disabled clusters, the other is not. We use the one that is not resized, which is probably wrong.
